tools/simplesimClient/main.cpp

```diff
@@ -94,6 +99,8 @@ int main( int argc, char * argv[] )
         settingsCmdPub->Publish( settingsCmd );
     } );
 
+    app.exec();
```

Comment:
This blocks until the Qt window is closed, so the message subscriber never runs.